### ISSUE TITLE
Monorepo implicit `tsconfig-paths` / `tsconfig.json` resolution issue

### ISSUE THREAD

**Issue by eliellis** [wallabyjs/quokka#816]
### Issue description or question

Hi guys, just bought Quokka Pro to make use of local project imports and immediately ran into an issue with tsconfig path aliasing. 

Given a standard Nx monorepo setup, wherein there is no "root level" tsconfig file (`tsconfig.base.json` only), but there *are* individual project-level `tsconfig.json` files, it appears as if Quokka is unable to resolve the correct `tsconfig.json` path (and subsequently the aliases in `compilerOptions.paths`) when the evaluated file is a sibling or child to a valid `tsconfig.json` file, e.g. `@test-quokka-workspace/library-b` is not resolvable by default from `libs/library-a/quokka.ts` in the below structure with Quokka:

```
test-quokka-workspace/
├─ apps/
├─ libs/
│  ├─ library-a/
│  │  ├─ tsconfig.json
│  │  ├─ quokka.ts
│  ├─ library-b/
│  │  ├─ tsconfig.json
├─ tsconfig.base.json

```

Given that `quokka.ts` is a sibling to a valid `tsconfig.json` file extending from `tsconfig.base.json` (which contains the path aliases) I would expect this to set everything up correctly. 

Running `quokka.ts` from the root of the project with the following works just fine: `TS_NODE_PROJECT="libs/library-a/tsconfig.json" npx ts-node -r tsconfig-paths/register ./libs/library-a/quokka.ts`. 

I tried setting the appropriate Quokka config to force the `TS_NODE_PROJECT` environment variable to get picked up as `libs/library-a/tsconfig.json` and after some fiddling I was able to get everything working by setting `TS_NODE_PROJECT` to the *absolute* path of the desired `tsconfig.json` file, which appears to be a half-decent workaround but is otherwise a strange endeavor to get something like this working. 

Hoping you guys can take a look!

Is this issue related to Quokka not outputting the expected results of your code?: Yes

### Sample repository link
See [here](https://github.com/eliellis/test-quokka-workspace) for a simple repro. Specifically [this file](https://github.com/eliellis/test-quokka-workspace/blob/main/libs/library-a/quokka.ts).

### Quokka.js Console Output
```shell
Cannot find module '@test-quokka-workspace/library-b' 
Require stack: 
- ./libs/library-a/quokka.js 
  ​​​​​at ​​​​​​​​Module._compile​​​ ​internal/modules/cjs/loader.js:1085​
  ​​​​​at ​​​​​​​​Object.<anonymous>​​​ ​libs/library-a/quokka.ts:1​
  ​​​​​at ​​​​​​​​require​​​ ​internal/modules/cjs/helpers.js:92​
  ​​​​​at ​​​​​​​​Function.Module._load​​​ ​internal/modules/cjs/loader.js:746​
  ​​​​​at ​​​​​​​​Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename]​​​ ​./node_modules/@cspotcode/source-map-support/source-map-support.js:811​
  ​​​​​at ​​​​​​​​Function.Module._resolveFilename​​​ ​internal/modules/cjs/loader.js:902​
​​​​​​
​​​Couldn't find tsconfig.json. tsconfig-paths will be skipped​​​
```
### Code editor version

Visual Studio Code v1.71.0

### OS name and version

macOS 12.3.1

**Comment by smcenlly** [comment]
Thanks for providing the sample repo!

Quokka has first class support for mono-repos that use `package.json` but the NX workspace is a little different as project structure isn't as explicit on disk. Quokka supports NX workspaces but unfortunately `tsconfig-paths` does not and Quokka was not explicitly passing the resolved TypeScript configuration.

We have updated the latest version of Quokka to support your project structure. Please update to `Quokka for VS Code `v1.0.503` and everything should work for you.

**Comment by eliellis** [comment]
Thank for getting to this so quickly! I just pulled the latest, but it appears to still be having trouble fully resolving paths even with the update. Adding `library-c`, and importing via alias in `library-b`, then running Quokka on the same `qoukka.ts` file as before, I am seeing:

```
⨯ Unable to compile TypeScript: 
libs/library-b/src/lib/library-b.ts(1,26): error TS2307: Cannot find module '@test-quokka-workspace/library-c' or its corresponding type declarations. 
```

Please refer to the example repo for a bit more context. Again, thanks so much for the great work. 🙂

**Comment by smcenlly** [comment]
@eliellis - sorry you ran into this issue. There was one more problem that was affecting our production build but not our dev/testing environment related to `ts-node` this time. This has been fixed in the latest version of Quokka, `v1.0.504`
